docs/maps/asset-tracking-tutorial.asciidoc

```diff
@@ -8,7 +8,7 @@ In this tutorial, you’ll look at live urban transit data from the city of Port
 
 You’ll learn to:
 
-- Use Logstash to ingest the TriMet REST API into Elasticsearch.
+- Use Filebeat to ingest the TriMet REST API into Elasticsearch.
```

Comment:
Use this shared attribute to refer to Filebeat throughout the doc.

```suggestion
- Use {filebeat} to ingest the TriMet REST API into Elasticsearch.
```

Reply:
Ahh wicked I didnt know this was a thing, thank you